cypress/support/pages/admin_console/manage/realm_settings/RealmSettingsPage.ts

```diff
@@ -1,5 +1,6 @@
 export default class RealmSettingsPage {
-    saveBtn: string;
+    generalSaveBtn: string;
```

Comment:
These field declarations can be initialized directly so they do not have to be assigned in the constructor. For example:

```suggestion
    generalSaveBtn = "general-tab-save"
```